.github/assets/js/command-palette.js

```
@@ -93,22 +123,105 @@ function renderSections(sections, container) {
+        if (iconStr.startsWith('<i') && 
+            (/class=\"(fa-|ai )/.test(iconStr) || iconStr.includes('fa-solid') ||
+             iconStr.includes('fa-brands') || iconStr.includes('fa-regular') ||
+             iconStr.includes('ai ai-'))) {
+          // Adding extra validation for classes
+          const validIconPattern = /^<i class=\"(fa|fas|far|fal|fab|ai)[ -][\w\d -]+"><\/i>$/;
+          if (validIconPattern.test(iconStr)) {
+            iconEl.innerHTML = sanitizeHTML(iconStr);
+          } else {
+            // Fallback to displaying as text if pattern doesn't match exactly
+            iconEl.textContent = '🔍'; // Default search icon as fallback
+            if (CONFIG && CONFIG.DEBUG) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)